client/src/components/FormField/RegisteredMFAMethodListField/RegisteredMFAMethodListField.scss

```diff
@@ -1,9 +1,40 @@
 .registered-mfa-method-list-field {
   padding: $input-padding-y 0;
+
+  &__button, &__no-methods {
+    margin-bottom: $input-padding-y-lg;
+  }
 }
 
 .method-list {
   list-style: none;
   margin: 0;
   padding: 0;
 }
+
+.registered-method-list-item {
+  margin-bottom: $input-padding-y-lg;
+
+  &__control {
+    border: 0;
+    margin: 0;
+    padding: 0;
+    background: none;
+    text-decoration: none;
+    color: $link-color;
+    cursor: pointer;
+
+    &:hover {
+      color: $link-hover-color;
+      text-decoration: underline;
+    }
+
+    & + & {
+      margin-left: $input-padding-x-lg;
+    }
+  }
+}
+
+.registered-mfa-method-list-field-register-modal {
+  height: initial;
```

Comment:
@robbieaverill This is a fix for that weird overflow issue you pointed out to me on Slack.

Reply:
Fancy adding a comment to say that's what it's for?